### PR TITLE
add Openup

### PR DESCRIPTION
**Pull request by Horryportier**
Add openup project 
[repo](https://github.com/Horryportier/openup)
This project uses:
-Bubbletea{list,textinput}
-Lipgloss

### CODE REVIEW

**Comment by Horryportier** [comment]
sorry for this wired pr i don't know what  I'm doing

**Comment by bashbunni** [comment]
Some feedback: I think the project should be maybe more idiomatic with Go. Users should be able to run `go build` and `go install` to install and build the project instead of relying on shell scripts. Because of this I think a lot of our user base won't take the extra steps to try it out for themselves. Thoughts?

**Comment by Horryportier** [comment]
Thanks for mentioning it. I'm an total noobie in using go so i didn't even know it's possible. I will for sure check it out. 😄

**Comment by Horryportier** [comment]
> Some feedback: I think the project should be maybe more idiomatic with Go. Users should be able to run `go build` and `go install` to install and build the project instead of relying on shell scripts. Because of this I think a lot of our user base won't take the extra steps to try it out for themselves. Thoughts?

I've redone installation process to use `go install`. Here's the new [README](https://github.com/Horryportier/openup/blob/main/README.md) with instructions.

File: bubbletea/README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Bubble Tea Examples
 
 ## Difficulty: Easy
+### [Openup](https://github.com/Horryportier/openup)
+
+Tui for opening files from anywhere.
```
Comment (maaslalani) [inline bubbletea/README.md:6]:
```suggestion
TUI for opening files from anywhere.
```

**Review by maaslalani — APPROVED** [review]

**Comment by maaslalani** [comment]
Awesome work @Horryportier, this is great! I love the GIF in the README as well. @bashbunni is this good to merge?

**Pull request merged**